includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -523,8 +523,12 @@ public function javascript_params() {
 		// Amazon Pay feature flag.
 		$stripe_params['isAmazonPayAvailable'] = WC_Stripe_Feature_Flags::is_amazon_pay_available();
 
+		$is_add_payment_method               = is_wc_endpoint_url( 'add-payment-method' );
```

PHPMD (2.15.0)

[error] The variable $is_add_payment_method is not named in camelCase. (undefined)

(CamelCaseVariableName)